### PR TITLE
[sailfish-browser] Add touch-related test pages. Contributes to JB#50875

### PR DESCRIPTION
**Pull request by llewelld**
Adds two new test pages:

1. Touch action test page, for testing the touch-action css property.
2. Touch/click test page. On desktop clicking on the items will generate
mouse but not touch events. On mobile pressing the items should trigger
both touch and mouse events.

### CODE REVIEW

**Comment by llewelld** [comment]
I forgot whether this should be merged into master or sailfishos-esr60. If it's the former I'll rebase.

File: tests/manual/testpage.html [hunk 1]
```diff
@@ -143,6 +143,14 @@
       <a href="test-fullscreen-video-l.html">Long page containing embeded video</a>
     </div>
 
+    <div class="header"><h2>Touch input</h2></div>
+    <div>
+      <a href="testtouchaction.html">Touch actions and the touch-action css property</a>
+    </div>
+    <div>
+      <a href="testtouchclick.html">Touch input vs. mouse input</a>
+    </div>
```
Comment (rainemak) [inline tests/manual/testpage.html:152]: Thank you for doing this part as well! Me likes!

**Review by rainemak — APPROVED** [review]
LGTM

**Comment by rainemak** [comment]
While you're checking this maybe you'd like check https://github.com/sailfishos/sailfish-browser/pull/705 as well.

**Comment by llewelld** [comment]
Changed to merge into next branch.

**Comment by llewelld** [comment]
Rebased on top of next.

**Pull request merged**